selfdrive/athena/tests/test_athenad.py

```diff
@@ -34,6 +35,13 @@ def tearDown(self):
     athenad.upload_queue = queue.Queue()
     athenad.cur_upload_items.clear()
 
+    for i in os.listdir(athenad.ROOT):
+      p = os.path.join(athenad.ROOT, i)
+      if os.path.isdir(p):
+        shutil.rmtree(p)
+      else:
+        os.unlink(p)
+
```

Comment:
I put this stuff in `setUp` because then if a test fails I can still inspect the state of things, is there some advantage to putting it in `tearDown`?  I just always felt better putting this kind of stuff in setup to ensure tests always start clean, too (would rather have exceptions in this code happen in `setUp`).

Reply:
Oh, didn't notice I used tearDown. It's fine to put it back!